pkg/generators/enum.go

```diff
@@ -102,7 +102,10 @@ func parseEnums(c *generator.Context) enumMap {
 				}
 			}
 		}
-		// find all enum values from constants, and try to match each with its type.
+	}
+
+	// find all enum values from constants, and try to match each with its type.
+	for _, p := range c.Universe {
```

Comment:
there is no difference to the previous code, is there?

Reply:
hmm, looks like you claim there is :) Order changes. Constants can be defined before their type. Makes sense.